remotesigning/remote_signing.go

```diff
@@ -56,10 +56,9 @@ func GraphQLResponseForRemoteSigningWebhook(
 	webhook webhooks.WebhookEvent,
 	seedBytes []byte,
 ) (SigningResponse, error) {
-	if !validator.ShouldSign(webhook) {
+	if !validator.ShouldSign(webhook, seedBytes) {
```

Comment:
The validator should not take a seed as a parameter, If we are only validating pubkeys, it should take the xpub as a parameter. The problem is that most of our customers cannot dump the private key value here. So it's actually a different services handling the private keys. 

I know that the L1 address is a hardened path so you'll probably need to validate each job individually.

Reply:
Just checking, do you want the customer to pass the base xpub? Or the child pubkeys?

Reply:
Probably the hardened xpub that we ask use for? like the root key for all hardened path

Reply:
Spoke with Alec about this: 

TLDR: We will keep the two separate derivation paths in the signable payload since we still need the full derivation_path to sign transactions later. I am renaming l1_destination_path to destination_derivation_path for more clarity and will add some comments to make it more clear.